pvlib/tests/iotools/test_panond.py

```diff
@@ -0,0 +1,243 @@
+"""
+test iotools for panond
+"""
+
+from pvlib.iotools import read_panond, parse_panond
+from pvlib.tests.conftest import DATA_DIR
+import io
+# Not sure if I am creating these test scenarios correctly
+
+fn_file = DATA_DIR / 'CPS SCH275KTL-DO-US-800-250kW_275kVA_1.OND'
+ond_file = read_panond(fn_file)
+
```

Comment:
Tests need to be defined in functions that start with `test_`. There needs to be at least one `assert` that checks some of the content read from the test file against the known values.

```suggestion
def test_read_ond_file():
    fn_file = DATA_DIR / 'CPS SCH275KTL-DO-US-800-250kW_275kVA_1.OND'
    ond_file = read_panond(fn_file)
    # expected
    f_obj = io.StringIO(fn_str)
    ond_str = parse_panond(f_obj)
    assert ond_file.keys() == ond_str.keys()
```